src/spinors.jl

```diff
@@ -39,6 +39,7 @@ function directsum(b1::SumBasis, b2::SumBasis)
     bases = [b1.bases...;b2.bases...]
     return SumBasis(shape, (bases...,))
 end
+directsum() = GenericBasis(0)
```

Comment:
Similar to the varargs issue: Varargs implies that it can have zero arguments, but the already defined method could not deal with zero arguments.